### DISCUSSION TITLE
Sync multiple emby accounts on one instance

### DISCUSSION THREAD

**Discussion by jnthnwrgg** [luigi311/JellyPlex-Watched#190] [Q&A]
I've been brought here via Reddit, and the possibility jellyplex-watched may resolve an issue for me.

I have one emby server with 3 user accounts; bob, charlie, and cinema. When bob and charlie watch anything together, they use the cinema account and manually mark content as watched on their own accounts.

Is it possible to use jellyplex to monitor the cinema account, and automatically update bob and charlies accounts when they watch something together,

**Comment by luigi311** [comment]
Right now theres no way to do from one account to multiple others due to how i structured the user list. I will need to redesign it and it is on my to do list.
